codegen/swift/swift-entities/src/main/resources/client/SwiftAPIClientCore.mustache

```diff
@@ -142,7 +142,7 @@ public extension Gateway {
                     observer.send(error: responseError)
                 }
             }
-            disposable.add {
+            disposable.observeEnded {
```

Comment:
is it backward compatible with the previous Swift versions?

Reply:
In order to support Swift 4, we've updated all related libraries, like Moya and so on. So, Moya has deprecated API add(). Instead, they've added this one.